spec/test/collections/test_djornl.py

```diff
@@ -58,7 +58,7 @@ def test_node(self, query_name=None, test_data=None):
             {
                 'data': {'_key': 'ABCDE', 'clusters': ['GO:0003700', 'GO:0003700']},
                 'valid': False,
-                'error': "\['GO:0003700', 'GO:0003700'\] has non-unique elements"
+                'error': "\\['GO:0003700', 'GO:0003700'\\] has non-unique elements"
```

Comment:
did this just start throwing errors? I wonder why it didn't before.

Reply:
I don't think there's an actual problem in runtime, but my editor's linter had a warning or error. The warning does make sense, because within double quotes you have escape characters, and `\[` is not a valid escape character. Python probably handles this by just ignoring the error and making it a literal.